From 67f83e8dc551d1850bc0a814d3c3a759b38c3858 Mon Sep 17 00:00:00 2001
From: Ryan <mukhery@gmail.com>
Date: Thu, 2 Jan 2025 20:13:48 -0500
Subject: [PATCH] apply fix from https://github.com/google/jsonnet/pull/1159

---
 python/_jsonnet.c | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/python/_jsonnet.c b/python/_jsonnet.c
index d509b72..0fca8cf 100644
--- a/python/_jsonnet.c
+++ b/python/_jsonnet.c
@@ -186,7 +186,7 @@ static struct JsonnetJsonValue *cpython_native_callback(
     }
 
     // Call python function.
-    result = PyEval_CallObject(ctx->callback, arglist);
+    result = PyObject_CallObject(ctx->callback, arglist);
     Py_DECREF(arglist);
 
     if (result == NULL) {
@@ -226,7 +226,7 @@ static int cpython_import_callback(void *ctx_, const char *base, const char *rel
 
     PyEval_RestoreThread(*ctx->py_thread);
     arglist = Py_BuildValue("(s, s)", base, rel);
-    result = PyEval_CallObject(ctx->callback, arglist);
+    result = PyObject_CallObject(ctx->callback, arglist);
     Py_DECREF(arglist);
 
     if (result == NULL) {
-- 
2.34.1

